### DISCUSSION TITLE
Linking error CoreML in version 2.0.0-alpha.4

### DISCUSSION THREAD

**Discussion by mapaaa** [pykeio/ort#143] [Q&A] [answered]
Hi! I am having a problem building using CoreML. I have this dependency in Cargo.toml:

`ort = { version="2.0.0-alpha.4", features=["coreml"] }`

When I do `cargo build` I get this strange linking error:
```
ld: Undefined symbols:
            ___isPlatformVersionAtLeast, referenced from:
                onnxruntime::coreml::util::HasRequiredBaseOS() in libonnxruntime.a[48](host_utils.mm.o)
                onnxruntime::coreml::Execution::LoadModel() in libonnxruntime.a[49](model.mm.o)
               onnxruntime::coreml::Execution::Predict(std::__1::unordered_map<std::__1::basic_string<char,....
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

What would be the cause for this?

**Comment by decahedron1** [comment]
What version of macOS are you running? Intel or Apple silicon?

**Reply by mapaaa** [reply]
I am running on Apple M1 and Version 14.2.1 macOS

**Reply by mapaaa** [reply, marked as answer]
After more search, I think  ___isPlatformVersionAtLeast comes from `libclang_rt.osx.a` which was not properly linked.
Did 
`RUSTFLAGS="-L /Library/Developer/CommandLineTools/usr/lib/clang/15.0.0/lib/darwin -l clang_rt.osx" cargo build` and it worked
